api/Element.json

```diff
@@ -5133,7 +5200,7 @@
               "notes": "This API was previously available on the <code>Node</code> API."
             },
             "webview_android": {
-              "version_added": true,
+              "version_added": "4.4.3",
```

Comment:
4.4.3 is being set for `localName`, `namespaceURI`, and `prefix`, the 3 attributes in https://github.com/mdn/browser-compat-data/issues/6545. This is the result of mirroring, and looks correct. We'll change these entries again later, but that's OK.